src/main.css

```diff
@@ -36,4 +37,19 @@
   .backface-visibility {
     backface-visibility: hidden;
   }
+
+  .custom-404::before {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    pointer-events: none;
+    background: radial-gradient(
+      circle 10vmax at var(--cursorX) var(--cursorY),
+      rgba(0, 0, 0, 0) 0%,
+      rgba(0, 0, 0, 0.5) 80%,
+      rgba(0, 0, 0, 0.95) 100%
+    );
+  }
```

Comment:
On peut styler les [pseudo-elements (:before, :after, ...)](https://tailwindcss.com/docs/hover-focus-and-other-states#pseudo-elements) avec Tailwind